mod/discussions/start.php

```diff
@@ -11,6 +11,12 @@
 function discussion_init() {
 
 	elgg_register_page_handler('discussion', 'discussion_page_handler');
+	
+	elgg_register_menu_item('site', [
```

Comment:
Broader discussion: https://elgg.org/discussion/view/2561039/should-site-menu-items-be-added-by-default